### PR TITLE
chore: Add Migrate test

### PR DESCRIPTION
**Pull request by rollandf**
Test clearing of State label on NV IPAM config map

### CODE REVIEW

**Review by ykulazhenkov — APPROVED** [review]

File: pkg/migrate/migrate_test.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	})
+	It("should not fail if state label does not exists", func() {
+		createConfigMap(false)
+		err := Migrate(goctx.Background(), testLog, k8sClient)
+		Expect(err).NotTo(HaveOccurred())
+	})
+	It("should not fail if NVIPAM CM does not exists", func() {
+		err := Migrate(goctx.Background(), testLog, k8sClient)
+		Expect(err).NotTo(HaveOccurred())
+	})
```
Comment (adrianchiris) [inline pkg/migrate/migrate_test.go:57]: can you add a fail test as well ?

File: pkg/migrate/suite_test.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+	By("bootstrapping test environment")
+
+	testEnv = &envtest.Environment{}
+
+	cfg, err := testEnv.Start()
+	Expect(err).NotTo(HaveOccurred())
+	Expect(cfg).NotTo(BeNil())
+
+	err = batchv1.AddToScheme(scheme.Scheme)
```
[the remark this answers is not in the document]
Comment (rollandf) [reply pkg/migrate/suite_test.go]: Done

File: pkg/migrate/migrate_test.go (hunk 1 above)
Comment (rollandf) [reply pkg/migrate/migrate_test.go:57]: Not easily with this framework.
The possible errors are on API calls.

Comment (adrianchiris) [reply pkg/migrate/migrate_test.go:57]: right ! need to use fake clientset.

**Review by adrianchiris — APPROVED** [review]

**Pull request merged**